gax-grpc/src/main/java/com/google/api/gax/grpc/OperationCallable.java

```diff
@@ -108,17 +118,18 @@
     if (context.getChannel() == null) {
       context = context.withChannel(clientContext.getChannel());
     }
-    ApiFuture<Operation> initialCallFuture = initialCallable.futureCall(request, context);
-    Duration pollingInterval =
-        settings != null ? settings.getPollingInterval() : OperationFuture.DEFAULT_POLLING_INTERVAL;
-    OperationFuture<ResponseT> operationFuture =
-        OperationFuture.create(
-            operationsClient,
-            initialCallFuture,
-            clientContext.getExecutor(),
-            responseClass,
-            pollingInterval);
-    return operationFuture;
+    return futureCall(initialCallable.futureCall(request, context));
+  }
+
+  OperationFuture<ResponseT> futureCall(ApiFuture<Operation> initialFuture) {
```

Comment:
There is already a futureCall override of this method, which accepts request & context (just above this method). This particular method is package-private. It exists because it is directly used from two other public methods: `futureCall(request, context)` and `resumeFuture()`. This method is not private, mostly because it is called once in test to test some unlikely cancellation/error cases.